scripts/docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM jupyter/base-notebook:python-3.11.6
+
+LABEL name="pymc-marketing"
+LABEL description="Environment for PyMC Marketing"
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# Switch to jovyan to avoid container runs as root
+USER $NB_UID
+
+COPY /scripts/docker/environment-dev.yml .
+RUN mamba env create -f environment-dev.yml && \
+    /bin/bash -c ". activate pymc-marketing-dev" && \
+    python -m ipykernel install --user --name=pymc-marketing-dev && \
+    conda clean --all -f -y
+
+#Setup working folder
+WORKDIR /home/jovyan/work
```

Comment:
Is this too specific?

Reply:
I think it is to make sure we have a clean working folder. Not a strong opinion here. I took it from the pymc one ;)

Reply:
Does jovyan mean anything?

Reply:
Default user name in docker-jupyter? COPY /scripts/docker/environment-dev.yml . 😆